### PR TITLE
Home, Nav, and Search updates

### PR DESCRIPTION
**Pull request by riazahmed01**
Changes:

- Improvements to home page
- Minimalistic search bar
- Improved nav and added small screen drop down nav

### CODE REVIEW

**Review by azuhayer — APPROVED** [review]

**Review by NikhilPrajith — CHANGES_REQUESTED** [review]
Doesn't the webpage seem a bit zoomed in? Maybe it's the font size and the navbar being too big

**Comment by NikhilPrajith** [comment]
Let me see if I can edit it

File: src/components/NavBar/NavBar.jsx
Comment (NikhilPrajith) [inline src/components/NavBar/NavBar.jsx:1]: Would it be better to leave the navbar as fixed to the screen?

Comment (NikhilPrajith) [reply src/components/NavBar/NavBar.jsx:1]: I currently made it as fixed!

**Review by NikhilPrajith — APPROVED** [review]

**Pull request merged**